tests/test_visualize.py

```diff
@@ -81,6 +81,19 @@ def test_confusion_matrix_plots_have_correct_data(self, classifier: Model):
         assert "Predicted Label" == ax.get_xlabel()
         plt.close()
 
+    def test_confusion_matrix_plots_have_correct_data_with_threshold(
```

Comment:
Maybe add a test for the `np.where` line? It's easier to parametrize it and pass edge cases